deploy/koku-migrations-cji.yaml

```diff
@@ -0,0 +1,65 @@
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: koku-migrations-cji
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: koku-migrations-cji
+  spec:
+    database:
+      sharedDbAppName: koku
+    dependencies:
+      - koku
+    envName: ${ENV_NAME}
+    jobs:
+    - name: script-runner-${IMAGE_TAG}
```

Comment:
This `job` name should be different than the `koku-cji` job.

Perhaps we can call this `migration-runner-${IMAGE_TAG}`?